src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java

```diff
@@ -37,15 +38,14 @@ public DayDiscountDTO getDayDiscountById(@PathVariable long id) {
     return dayDiscountService.findDayDiscountById(id);
   }
 
-  @GetMapping("/{startDate}/{endDate}")
-  public ResponseEntity<String> checkUniqueOfDayDiscount(@PathVariable long startDate,
-      @PathVariable long endDate) {
-    if (dayDiscountService.getDayDiscountsForPeriod(new Date(startDate), new Date(endDate)).size()
-        > 0) {
-      return new ResponseEntity<String>("No ok", HttpStatus.OK);
-    } else {
-      return new ResponseEntity<String>("ok", HttpStatus.OK);
-    }
+  @GetMapping("/{startDate}/{endDate}/{startTime}/{endTime}")

```

Comment:
http://stackoverflow.com/questions/13715811/requestparam-vs-pathvariable

And @RequestParam isn't required.